Makefile

```diff
@@ -12,7 +12,7 @@ SPEC=$(RPM_NAME).spec
 RPMFILE=$(shell rpmspec -D"VERSION $(VERSION)" -q packaging/$(SPEC).in).rpm
 APPSTREAMFILE=org.cockpit-project.$(PACKAGE_NAME).metainfo.xml
 VM_IMAGE=$(CURDIR)/test/images/$(TEST_OS)
-# stamp file to check if/when npm install ran
+# stamp file to check for node_modules/
```

Comment:
The commit message of "Makefile: Some noise cleanup" does not match the actual patch.

Reply:
Adjusted the body of the commit message.